README.md

```diff
@@ -1,8 +1,64 @@
+# Eclipse LSP4MP - Language Server for Eclipse MicroProfile
+
 [![Build Status](https://ci.eclipse.org/lsp4mp/buildStatus/icon?job=lsp4mp%2Fmaster)](https://ci.eclipse.org/lsp4mp/job/lsp4mp/job/master/)
+[![License](https://img.shields.io/badge/License-EPL%202.0-brightgreen.svg)](https://www.eclipse.org/legal/epl-2.0/)
+
+A language server implementation based on the [Language Server Protocol](https://github.com/Microsoft/language-server-protocol) for [Eclipse MicroProfile](https://microprofile.io/). This Language Server for Eclipse MicroProfile (LSP4MP) provides core language support capabilities (such as code complete, diagnostics, quick fixes) to enable developers to quickly and easily develop applications using MicroProfile APIs.
 
 This project contains:
 
- * [MicroProfile JDT LS Extensions](./microprofile.jdt)
- * [MicroProfile Language Server](./microprofile.ls)
+- [MicroProfile JDT LS Extensions](./microprofile.jdt)
+- [MicroProfile Language Server](./microprofile.ls)
+
+You can build all projects at once by running the `buildAll.sh` script (`buildAll.bat` on Windows).
+
+## Features
+
+#### Properties files
+
+In `microprofile-config.properties` files, you will benefit with:
+
+- Completion support for MicroProfile properties
+- Hover support for MicroProfile properties
+- Definition support for MicroProfile properties
+- Format support for MicroProfile properties
+- Validation and Quick Fix support for MicroProfile properties
+- Outline support (flat or tree view)
+
+#### Java files
+
+In Java files, you will benefit with:
+
+- Completion support for MicroProfile
+- Hover support for MicroProfile
+- Validation and Quick Fix support for MicroProfile
+- Code Lens support for MicroProfile
+- Code snippets
+
+## Demo
+
+TODO
+

```

Comment:
Maven corrdinate misses.

Maven coordinates:
------------------

Here are the Maven coordinates for LSP4MP (replace the `X.Y.Z` version with the [latest release](https://repo.eclipse.org/content/repositories/lsp4mp-releases)):
```xml
<dependency>
    <groupId>org.eclipse.lsp4mp</groupId>
    <artifactId>org.eclipse.lsp4mp</artifactId>
    <version>X.Y.Z</version>
    <!-- classifier:uber includes all dependencies -->
    <classifier>uber</classifier>
</dependency>
```

for Gradle:
```
compile(group: 'org.eclipse.lsp4mp', name: 'org.eclipse.lsp4mp', version: 'X.Y.Z', classifier: 'uber')
```

You will have to reference the Maven repository hosting the dependency you need. E.g. for Maven, add this repository to your pom.xml or settings.xml :
```xml
<repository>
  <id>lsp4mp-releases</id>
  <url>https://repo.eclipse.org/content/repositories/lsp4mp-releases/</url>
  <snapshots>
    <enabled>false</enabled>
  </snapshots>
  <releases>
    <enabled>true</enabled>
  </releases>
</repository>
```

And if you want to consume the SNAPSHOT builds instead:
```xml
<repository>
  <id>lsp4mp-snapshots</id>
  <url>https://repo.eclipse.org/content/repositories/lsp4mp-snapshots/</url>
  <releases>
    <enabled>false</enabled>
  </releases>
  <snapshots>
    <enabled>true</enabled>
  </snapshots>
</repository>
```